docs/_docs/triggers.md

```diff
@@ -47,35 +47,35 @@ Once the settings are added, click _Save and Continue_ to add the new trigger an
+The last part of adding a trigger is setting the API configuration.
 
-Zapier uses either a Polling API call to check periodically for new or updated data, or a REST hook with a subscription URL where Zapier can subscribe to receive new or updated data automatically. 
+Zapier uses either a Polling API call to check periodically for new or updated data, or a REST Hook with a subscription URL where Zapier can subscribe to receive new or updated data automatically.
 
-Polling API results are expected to be an array with 0 or more objects that will be passed to Zapier's [deduper](https://zapier.com/developer/documentation/v2/deduplication/).  The deduper will return any items that haven't been received before, and use them to run the subsequent steps in the user's Zap.
+Polling API results are expected to be an array with 0 or more objects that will be passed to Zapier's [deduper](https://platform.zapier.com/docs/dedupe). The deduper will return any items that haven't been received before, and use them to run the subsequent steps in the user's Zap.
 
 ### Polling Trigger
 
-Zapier selects a _Polling_ trigger by default, where Zapier will send a `GET` request to an API endpoint URL to request new data—and in live Zaps, Zapier automatically deduplicates and sorts for the newest data from the API. Live Zaps automatically poll the URL for new data every 1 to 15 minutes, depending on the user's Zapier plan (see [Plans & Pricing](https://zapier.com/pricing) for more details).
+Zapier selects a _Polling_ trigger by default, where Zapier will send a request to an API endpoint URL to request new data—and in live Zaps, Zapier automatically deduplicates and sorts for the newest data from the API. Live Zaps automatically poll the URL for new data every 1 to 15 minutes, depending on the user's Zapier plan (see [Plans & Pricing](https://zapier.com/pricing) for more details).
 
-To add a polling trigger, select _Polling_ at the top of the settings page, then enter your API URL in the _API Endpoint_ field. If your API URL requires specific data in the URL, enter the following text in the URL where your API expects that data, replacing `key` for the input field key with the relevant data from the input form you created before:
+To add a polling trigger, select _Polling_ at the top of the settings page, then enter your API URL in the _API Endpoint_ field. If your API URL requires specific data in the URL path, enter the following text in the URL where your API expects that data, replacing `key` with the input field key containing the relevant data from the input form you created in the previous step:
 
 {% raw %}`{{bundle.inputData.key}}`{% endraw %}
 
-Otherwise, Zapier will automatically send any other input field data in the request body with the API call.
+Otherwise, Zapier will automatically include any input field data with the API call as URL parameters (for GET requests) or in the request body (for POST requests, e.g. for GraphQL APIs).
```

Comment:
I don't think we do GraphQL by default. Suggested change:

```suggestion
Otherwise, Zapier will automatically encode any input field data with the API call as URL parameters (for GET requests) or in the request body with JSON (for POST requests).
```

Reply:
Yeah, that's a good point. I used that as an example because GraphQL is the most familiar type of API for me that might use POSTs for polling, but it does give the wrong impression about the level of support.

I'm curious about why you mention `encode` - is that just because we'll also default to URL-encoding the params? From the point of view of the developer, we're including them in the request and the encoding is transparent, so I'm not sure about the best word to use.